Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -5231,6 +5231,20 @@ ProcessSwapOnCrossSiteNavigationEnabled:
       "PLATFORM(PLAYSTATION)": false
       default: true
 
+PropagateDamagingInformation:
+   type: bool
+   status: testable
+   category: dom
+   humanReadableName: "Propagate Damaging Information"
+   humanReadableDescription: "Propagate Damaging Information"
```

Comment:
We should add `condition: USE(COORDINATED_GRAPHICS)` or platform GTK/WPE, but this setting is not used by other ports, no?